functions/starter-function/index.js

```diff
@@ -1,13 +1,39 @@
+// import middy from '@middy/core';
+// import jsonBodyParser from '@middy/http-json-body-parser';
+// import validator from '@middy/validator';
+// import httpErrorHandler from '@middy/http-error-handler';
 import { apiSuccess, apiFailure } from '@utils';
 import { getOrganizations } from '@services/github';
 
-exports.handler = async (event, _context, callback) => {
-	console.log(JSON.stringify(event));
+export const baseHandler = async (event, _context, callback) => {
 	try {
-		const response = await getOrganizations();
+		console.log('Inside baseHandler');
```

Comment:
Haha, I like this but let's make sure we don't commit these.

Reply:
Ok